src/services/tree.js

```diff
@@ -153,7 +153,14 @@ function sortNotes(parentNoteId, customSortBy = 'title', reverse = false, folder
             }
 
             function compare(a, b) {
-                return b === null || b === undefined || a < b ? -1 : 1;
+                if (!sortNatural){
+                    // alphabetical sort
+                    return b === null || b === undefined || a < b ? -1 : 1;
+                } else {
+                    // alphanumeric sort, or natural sort
+                    return a.localeCompare(b, undefined, {numeric: true, sensitivity: 'base'});
```

Comment:
I see a problem with not defining locale here - apparently in that case the system locale will be used, but that can be different than desired. Especially the problem can arise when you have a web version running on some linux host with locale you can't change - then your local desktop instance will "fight" with the server version over the sort order.

I'm thinking that for "permanent sorting" - by label - you should be able to define the locale. Maybe then rename the label to something like `#sortLocale=de`. For one time sorting I think it's acceptable without explicit locale, because then it makes sense it's done according to locale where you're executing the action.

Reply:
You concern makes sense. I've added the `#sortLocale` label.